lib/index.js

```diff
@@ -469,6 +471,55 @@ class AuthenticationPlugin {
     return this.getUsersRepository().delete(user._id, {refresh: 'wait_for'});
   }
 
+  /**
+   * Search users by ES query concerning exclusively their username
+   *
+   * @param {Object} searchBody - Search request body (query, sort, etc.)
+   * @param {Object} options - from (undefined), size (undefined)
+   *
+   * @returns {Promise.<{ hits, total }>}
+   */
+  async search (searchBody, { from, size } = {}) {
+    debug('exists(searchBody = %j)', searchBody);
+
+    const forbiddenProperties = [
+      'multi_match',
```

Comment:
Kuzzle already sanitizes queries and prevents multi-match ones. See here: https://github.com/kuzzleio/kuzzle/blob/master/lib/service/storage/elasticsearch.js#L106

Removing it allows to make the forbidden error below more consistent: otherwise, users cannot know if a query has been rejected because of a forbidden field, or because of a forbidden keyword.

Reply:
Kuzzle sanitizes only root key not nested ones, I've just tried with multi-match and it doesn't throw anything...

Reply:
You still have a good concern about the error message, I'll try to find a better solution